src/components/Button/Button.jsx

```diff
@@ -0,0 +1,9 @@
+function Button() {
+  return (
+    <div>
```

Comment:
no need for an empty div

Reply:
Fixed through deleting the Button component.